TNW responsive page clone with media queries

uryelah opened the pull request:
- layout with breakpoints for better cross-screen accessibility
- hover effects
- CSS only drop/hidden nav behavior
- semantic HTML
- responsive grid layout
- type-setting and use of external fonts and icons
- color variables

maelfosso commented on index.html:
```diff
@@ -0,0 +1,1010 @@
+  </nav>
+  <div class=" white-smudge media-query-1023 nav-patch">
+  </div>
+  <a class=" media-query-767 nav-logo nav-logo-outside crazy-pumpkin media-query-767" href="#"><img src="assets/img/tnw.svg" alt="logo" /></a>
```
I suggest you put it into the header or create a big section for that

uryelah replied:
From which and to which line?

maelfosso commented on index.html:
```diff
@@ -0,0 +1,1010 @@
+  <a class="nav-logo" id="nav-out" href="#"><img src="assets/img/tnw.svg" alt="logo" /></a>
+  <button class="ico-button-out">
+    <i class="fas fa-times"></i>
+  </button>
```
Please, correct that part.
Follow the suggestion left up

uryelah replied:
@maelfosso
That's interesting, we didn't get this bug on firefox

uryelah replied:
Oh you mean the animation flicker at the opening of the page? Please confirm that's the bug.

uryelah replied:
Huh, just display:none will not be enough. It will kill the transformation transition. It will need to be an animation then to make it still work.

uryelah replied:
Ok, so about the dividing the code in sections.
The whole body, outside of the hidden navs is inside a single div, we can change it to be a section, but that's necessary in order for sliding nav animation to work and it's the same in the actual site.
I think we can group all the hidden nav elements in a section though.

uryelah commented:
Sliding nav menu bug fixed

uryelah commented:
- Fixed Chrome sliding nav bug
- Fixed Article overlapping text at media query 767px bug
- In order to achieve sliding nav effect with CSS only, on mobile kept page content, minus headers and navs, wrapped on a single tag
- For the sliding nav effect to work using the `:checked ~ sibling` CSS trick it was necessary to keep the labels and input[type="checkbox"] outside any wrapping elements and as previous siblings of the hidden navigation menus and site wrapping
